src/init.cpp

```diff
@@ -155,6 +155,11 @@ static const char* DEFAULT_ASMAP_FILENAME="ip_asn.map";
  * The PID file facilities.
  */
 static const char* BITCOIN_PID_FILENAME = "bitcoind.pid";
+/**
+ * Whether this process has created a PID file.
+ * Used when determining whether we should remove the PID file on shutdown.
+ */
```

Comment:
nit: I think this more clearly explains the rationale without being too explicit about implementation
```suggestion
//! True if this process created the pid file, so we don't delete it too early if multiple processes were started
 */
```